.github/workflows/build-image.yml

```diff
@@ -0,0 +1,23 @@
+name: Build image and publish to Docker Hub
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+
+  deploy:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Login to Docker Hub
+      run: echo ${{ secrets.DOCKER_HUB_PASSWORD }} | docker login -u ${{ secrets.DOCKER_HUB_USERNAME }} --password-stdin
+    
+    - name: Build and push image
+      # Github stores the current tag in an enviroment variable (GITHUB_REF) in the format /refs/tags/TAG_NAME. 
+      # Using shell parameter expansion, we extract the TAG_NAME.
+      run: ./docker/push.sh 'prod' "${GITHUB_REF/refs\/tags\//}"
```

Comment:
I opened a ticket about improving push.sh to make it fail properly if any intermediate process fails (e.g. don't push if the build fails). Maybe we could include that here? (although I guess the push will fail also if the build fails, because the tagged version won't exist)